### PR TITLE
add properties to support devops server

### PR DESCRIPTION
**Pull request by AndrejLavrinovic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -170,7 +170,9 @@ Parameter | Description | Required? | SCM Specific? | Default
 **`repository`** | The repository to push SwaggerHub changes to | yes | - | - 
 **`repositoryOwner`** | The SCM account which owns the above repository | no | `GITHUB`. `BITBUCKET` | - 
 **`scmProject`** | Team Project which contains the target repository | no | `AZURE_DEVOPS_SERVICES` | - 
-**`scmOrganization`** | Specific to Azure DevOps Services, the organization with which to synchronize | no | `AZURE_DEVOPS_SERVICES` | - 
+**`scmOrganization`** | Specific to Azure DevOps Services, the organization with which to synchronize | no | `AZURE_DEVOPS_SERVICES` | -
+**`scmUrl`** | Host URL of the Azure DevOps Server | no | `AZURE_DEVOPS_SERVER` | -
```
Comment (AndrejLavrinovic) [inline README.md]: @michaelmelodysmartbear : Can shorten this to host URL of the SCM

Done

File: README.md [hunk 2]
```diff
@@ -170,7 +170,9 @@ Parameter | Description | Required? | SCM Specific? | Default
...
+**`scmProjectCollection`** | Project collection which contains the target repositories project | no | `AZURE_DEVOPS_SERVER` | DefaultCollection
```
Comment (AndrejLavrinovic) [inline README.md]: @michaelmelodysmartbear : CAn shorten to Project collection which contains the target repositories project.

Done

File: src/test/java/io/swagger/swaggerhub/plugin/utils/SwaggerHubUploadTestConstants.java [hunk 3]
```diff
@@ -26,6 +26,8 @@
     public static final String SCM_ACCOUNT= "scm-account";
     public static final String SCM_PERSONAL_ACCESS_TOKEN = "scm-personal-access-token";
     public static final String SCM_PROJECT = "scm-project";
+    public static final String SCM_URL = "http://localhost:1234";
+    public static final String SCM_PROJECT_COLLECTION = "DefaultCollection";
```
Comment (AndrejLavrinovic) [inline src/test/java/io/swagger/swaggerhub/plugin/utils/SwaggerHubUploadTestConstants.java]: @michaelmelodysmartbear : Remove references to TFS in the constant property names. We may have other SCM integrations in future which use those same properties

Done

File: src/test/java/io/swagger/swaggerhub/plugin/SwaggerHubUploadTest.java [hunk 4]
```diff
@@ -294,6 +290,64 @@ public void testMultiDefinitionsAreUploaded_andSCMSaveRequestMadeWithAccountPATA
...
+
+    @Test
+    public void testInputFileIsUploaded_andSCMSaveRequestMadeWithUrlAndProjectCollection() throws Exception {
+        //GIVEN
+        UrlPathPattern saveDefinitionRequest = stubSaveDefinitionRequest(API_OWNER, INPUT_FILE_API, INPUT_FILE_API_VERSION, IS_PRIVATE, OAS2, JSON, SWAGGERHUB_API_TOKEN);
+        UrlPathPattern saveSCMPluginConfigurationRequest = stubSaveSCMPluginConfigurationRequest(API_OWNER, INPUT_FILE_API, INPUT_FILE_API_VERSION, OAS2, SWAGGERHUB_API_TOKEN);
+
+        RequestPatternBuilder saveSCMConfigRequest = createPutSCMConfigRequestPatternWithUrlAndProjectCollection(API_OWNER, INPUT_FILE_API, INPUT_FILE_API_VERSION, OAS2, SCM_BRANCH,
+                SCM_ENABLE_INTEGRATION, SCM_REPOSITORY, SCM_REPOSITORY_OWNER, INPUT_FILE_FILENAME,
+                DefinitionFileFormat.JSON.getLanguageTarget(), TEST_RESOURCES_DIRECTORY, SCM_PERSONAL_ACCESS_TOKEN, SCM_PROJECT, SCM_URL, SCM_PROJECT_COLLECTION);
```
Comment (AndrejLavrinovic) [inline src/test/java/io/swagger/swaggerhub/plugin/SwaggerHubUploadTest.java]: @michaelmelodysmartbear : Remove references to TFS in the constant property names. We may have other SCM integrations in future which use those same properties

done

File: src/test/java/io/swagger/swaggerhub/plugin/SwaggerHubUploadTest.java [hunk 5]
```diff
@@ -294,6 +290,64 @@ public void testMultiDefinitionsAreUploaded_andSCMSaveRequestMadeWithAccountPATA
...
+                SCM_PROJECT, SCM_URL, SCM_PROJECT_COLLECTION);
+        RequestPatternBuilder saveSCMPluginRequestPattern2 = createPutSCMConfigRequestPatternWithUrlAndProjectCollection(API_OWNER, MULTI_UPLOAD_API_2_TITLE, MULTI_UPLOAD_API_2_VERSION, OAS2, SCM_BRANCH,
+                SCM_ENABLE_INTEGRATION, SCM_REPOSITORY, SCM_REPOSITORY_OWNER, MULTI_UPLOAD_API_2_FILENAME, DefinitionFileFormat.JSON.getLanguageTarget(), FILE_FINDER_DIRECTORY, SCM_PERSONAL_ACCESS_TOKEN,
+                SCM_PROJECT, SCM_URL, SCM_PROJECT_COLLECTION);
+        RequestPatternBuilder saveSCMPluginRequestPattern3 = createPutSCMConfigRequestPatternWithUrlAndProjectCollection(API_OWNER, MULTI_UPLOAD_API_3_TITLE, MULTI_UPLOAD_API_3_VERSION, OAS3, SCM_BRANCH,
+                SCM_ENABLE_INTEGRATION, SCM_REPOSITORY, SCM_REPOSITORY_OWNER, MULTI_UPLOAD_API_3_FILENAME, DefinitionFileFormat.YAML.getLanguageTarget(), FILE_FINDER_DIRECTORY, SCM_PERSONAL_ACCESS_TOKEN,
+                SCM_PROJECT, SCM_URL, SCM_PROJECT_COLLECTION);
```
Comment (AndrejLavrinovic) [inline src/test/java/io/swagger/swaggerhub/plugin/SwaggerHubUploadTest.java]: @michaelmelodysmartbear : Remove references to TFS in the constant property names. We may have other SCM integrations in future which use those same properties

Done

File: README.md [hunk 6]
```diff
@@ -212,13 +214,14 @@ App password permissions required are:
...
+* `AZURE_DEVOPS_SERVICES` and `AZURE_DEVOPS_SERVER` use personal access tokens which will eventually expire. In the event of token expiring, the integration will need to be deleted and recreated (either manually or via the plugin).
 
     See Microsoft documentation here for details on creating a Personal Access Token. Please ensure that the token has at minimum the scope Code (read and write) for the organization you wish to access
 
 * A list of `AZURE_DEVOPS_SERVICES` organizations that you currently have access to can be found [here](https://aex.dev.azure.com/me). Also listed are the projects for each organization.
 
-    Enter just the organization, without ‘dev.azure.com’ at the beginning or ‘.visualstudio.com’ at the end. For example, if your organization is ‘dev.azure.com/example-user’ or 'example-user.visualstudio.com’, enter just 'example-user’. 
+    Enter just the organization, without ‘dev.azure.com’ at the beginning or ‘.visualstudio.com’ at the end. For example, if your organization is ‘dev.azure.com/example-user’ or 'example-user.visualstudio.com’, enter just 'example-user’.
+
```
Comment (michaelmelodysmartbear) [inline README.md]: Remove whitespace
